src/panels/lovelace/cards/hui-picture-glance-card.ts

```diff
@@ -35,6 +35,23 @@ const STATES_OFF = new Set(["closed", "locked", "not_home", "off"]);
 
 @customElement("hui-picture-glance-card")
 class HuiPictureGlanceCard extends LitElement implements LovelaceCard {
+  public static async getConfigElement(): Promise<LovelaceCardEditor> {
+    await import(/* webpackChunkName: "hui-picture-glance-card-editor" */ "../editor/config-elements/hui-picture-glance-card-editor");
+    return document.createElement("hui-picture-glance-card-editor");
+  }
+  public static getStubConfig(): object {
+    return {
+      entity: "",
+      image:
+        "https://www.home-assistant.io/images/merchandise/shirt-frontpage.png",
+      camera_view: "auto",
```

Comment:
Only entities is required, so let's just have that in.

Reply:
And image I guess

Reply:
sure, there are some that are optional, but they do have defaults was my thinking

Reply:
The defaults are handled by the editor and code, no need to specify them.

Reply:
if done in the editor, it will get a bit more messy; you don't want defaults being populated if you are using a camera instead of an image, for example

Reply:
I don't understand what you mean? The result will be the same when you specify them here or don't specify them at all right?